buildSrc/build.gradle

```diff
@@ -191,6 +191,7 @@ if (project != rootProject) {
     distribution project(':distribution:archives:linux-tar')
     distribution project(':distribution:archives:windows-zip')
 
+    integTestRuntimeOnly(project(":libs:opensearch-common"))
```

Comment:
Sorry @nknize, referring to your previous comment:

> In the refactor effort :libs:opensearch-core will be pulling in dependencies not needed for :buildtools (e.g., lucene, log4j). We need a library like :opensearch-commons for classes that are dependency free for any other libraries that need foundation classes (e.g., PathUtils) without pulling in the monolith of :libs:core (currently :server).

But I see we still need both, why is that? I expected to `integTestRuntimeOnly(project(":libs:opensearch-common"))` only, what is missed to make it happen?

Reply:
Because the remaining `internal` classes in core are still needed by `build-tools`. I opted not to bring them over in this PR and instead move them in a follow up after the refactoring x-content PR. Really for no other reason than to avoid having an empty project.